tests/stress/conftest.py

```diff
@@ -17,22 +17,27 @@ def get_function_conpleteness_level(pytestconfig):
     return pytestconfig.getoption("--completeness_level")
 
 
-@pytest.fixture(scope="module", autouse=True)
-def set_polling_interval(duthost):
-    wait_time = 2
-    duthost.command("crm config polling interval {}".format(CRM_POLLING_INTERVAL))
-    logger.info("Waiting {} sec for CRM counters to become updated".format(wait_time))
-    time.sleep(wait_time)
-
-    yield
-
-    duthost.command("crm config polling interval {}".format(CRM_DEFAULT_POLL_INTERVAL))
-    logger.info("Waiting {} sec for CRM counters to become updated".format(wait_time))
-    time.sleep(wait_time)
+# @pytest.fixture(scope="module", autouse=True)
+# def set_polling_interval(duthost):
+#     wait_time = 2
+#     duthost.command("crm config polling interval {}".format(CRM_POLLING_INTERVAL))
+#     logger.info("Waiting {} sec for CRM counters to become updated".format(wait_time))
+#     time.sleep(wait_time)
+#
+#     yield
+#
+#     duthost.command("crm config polling interval {}".format(CRM_DEFAULT_POLL_INTERVAL))
+#     logger.info("Waiting {} sec for CRM counters to become updated".format(wait_time))
+#     time.sleep(wait_time)
```

Comment:
The `set_polling_interval` fixture doesn't seem to get used in the `stress` test module, so I commented it out to avoid confusion.